src/plugins/data_views/server/rest_api_routes/public/has_user_data_view.ts

```diff
@@ -88,7 +88,3 @@ const hasUserDataViewRouteFactory =
 export const registerHasUserDataViewRoute = hasUserDataViewRouteFactory(
```

Comment:
We could consider removing this route. I think its usage was replaced by the hasData service. There aren't any public docs. I don't see why this functionality needs to be exposed as a REST endpoint.